src/cpuinfo_mips.c

```diff
@@ -24,7 +24,7 @@ DECLARE_SETTER(MipsFeatures, eva)
 
```

Comment:
Should you declare a:
```c
DECLARE_SETTER(MipsFeatures, r6)
DECLARE_SETTER(MipsFeatures, crc32)
```
or in a later commit ?

Reply:
Yes I plan to update the implementation in another commit to separate bug fixes from enhancements.